### ISSUE TITLE
Pretrained resnext-101-64f-kinetics model

### ISSUE THREAD

**Issue by leonelcuevas** [kenshohara/3D-ResNets-PyTorch#35]
Hi,

I have seen you uploaded pretrained models using 64 frames for ucf101 and hmdb51. 
Is there a chance you can upload a 64f model pretrained using kinetics only? 

I would like to compare results between 16f and 64f, but in order to make a proper comparison I would rather use the pretrained model only on kinetics as well.

Thanks for the great work!

**Comment by kenshohara** [comment]
OK, I will upload the model later.
Please wait a moment.

**Comment by leonelcuevas** [comment]
Thank you very much!

**Comment by kenshohara** [comment]
I uploaded the file ```resnext-101-64f-kinetics.pth```.

**Comment by leonelcuevas** [comment]
Thank you!

**Comment by Fazlik995** [comment]
@kenshohara 

I can't get the same accuracy in the paper using  resnext-101-64f-kinetics.pth.

val.log accuracy about 85~86%

I use below to evaluate:

python main.py --root_path ~/project/ucf101 --video_path jpg0 --annotation_path ucf101_01.json
--result_path results/333 --dataset ucf101 --n_classes 101
--model resnext --model_depth 101 --resnet_shortcut B --resnext_cardinality 32 --sample_duration 16
--resume_path results/save_200.pth --batch_size 64 --n_threads 12 --no_train --no_val --test --test_subset val

Could you tell me what is my mistake?

video accuraccy is about 88%.
